CommonMark/Formatters/HtmlFormatter.cs

```diff
@@ -217,6 +217,29 @@ protected virtual void WriteBlock(Block block, bool isOpening, bool isClosing, o
                     break;
 
                 case BlockTag.IndentedCode:
+
+                    if (isOpening)
+                    {
+                        EnsureNewLine();
+
+                        Write("<pre><code");
+                        if (Settings.TrackSourcePosition)
+                            WritePositionAttribute(block);
+
+                        Write('>');
+                    }
+
+                    if (0 == (_settings.AdditionalFeatures & CommonMarkAdditionalFeatures.EmphasisInIndentedCode))
```

Comment:
all other places are using Settings instead of _settings. Not really important, just consistence. Though I am surprised that it is not prefixed with `this.` everywhere...